src/Sylius/Bundle/ApiBundle/Controller/UploadAvatarImageAction.php

```diff
@@ -57,7 +57,7 @@ public function __invoke(Request $request): ImageInterface
         $image->setFile($file);
 
         /** @var string $ownerIri */
-        $ownerIri = $request->get('owner');
+        $ownerIri = $request->request->get('owner');
```

Comment:
This could be considered as BC-Break, but I'm fine with that. Can we add a note about it?

Reply:
we have `@experimental` on the entire apiBundle so IMO it isn't BC-break, but ofc I will add a note to apiUpgrade file